<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>üí≠ Dreamcore Queue üí≠</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QListWidget" name="listQueue">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
        QListWidget {
            background-color: #e6e6fa;
            color: #9370db;
            border: 2px solid #d8bfd8;
            border-radius: 12px;
            padding: 10px;
        }
       </string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="inputLayout">
      <item>
       <widget class="QLineEdit" name="lineEditValue">
        <property name="styleSheet">
         <string notr="true">
          QLineEdit {
              border: 2px solid #b0c4de;
              border-radius: 10px;
              padding: 6px;
              background-color: #f0f8ff;
              color: #6a5acd;
              font-weight: bold;
          }
         </string>
        </property>
        <property name="placeholderText">
         <string>–í–≤–µ–¥–∏—Ç–µ —á–∏—Å–ª–æ üí´</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonPush">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                                stop:0 #b0e0e6, stop:1 #dda0dd);
              color: #4b0082;
              font-weight: bold;
              padding: 6px 12px;
              border-radius: 12px;
              border: 1px solid #d8bfd8;
          }
          QPushButton:hover {
              background-color: #e0ffff;
          }
         </string>
        </property>
        <property name="text">
         <string>Push üå†</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonPop">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                                stop:0 #b0e0e6, stop:1 #dda0dd);
              color: #4b0082;
              font-weight: bold;
              padding: 6px 12px;
              border-radius: 12px;
              border: 1px solid #d8bfd8;
          }
          QPushButton:hover {
              background-color: #e0ffff;
          }
         </string>
        </property>
        <property name="text">
         <string>Pop üíî</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="infoLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelSize">
        <property name="styleSheet">
         <string notr="true">
          QLabel {
              color: #6a5acd;
              font: bold 14pt &quot;Segoe UI&quot;;
              background-color: #f8f8ff;
              padding: 4px;
              border-radius: 6px;
          }
         </string>
        </property>
        <property name="text">
         <string>Size: 0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelEmpty">
        <property name="styleSheet">
         <string notr="true">
          QLabel {
              color: #6a5acd;
              font: bold 14pt &quot;Segoe UI&quot;;
              background-color: #f8f8ff;
              padding: 4px;
              border-radius: 6px;
          }
         </string>
        </property>
        <property name="text">
         <string>Empty: true</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFront">
        <property name="styleSheet">
         <string notr="true">
          QLabel {
              color: #6a5acd;
              font: bold 14pt &quot;Segoe UI&quot;;
              background-color: #f8f8ff;
              padding: 4px;
              border-radius: 6px;
          }
         </string>
        </property>
        <property name="text">
         <string>Front: -</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelBack">
        <property name="styleSheet">
         <string notr="true">
          QLabel {
              color: #6a5acd;
              font: bold 14pt &quot;Segoe UI&quot;;
              background-color: #f8f8ff;
              padding: 4px;
              border-radius: 6px;
          }
         </string>
        </property>
        <property name="text">
         <string>Back: -</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="taskLayout">
      <item>
       <widget class="QPushButton" name="buttonTask">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                                stop:0 #b0e0e6, stop:1 #dda0dd);
              color: #4b0082;
              font-weight: bold;
              padding: 6px 12px;
              border-radius: 12px;
              border: 1px solid #d8bfd8;
          }
          QPushButton:hover {
              background-color: #e0ffff;
          }
         </string>
        </property>
        <property name="text">
         <string>Task üíº</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textTask">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>55</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>55</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
          QTextEdit {
              background-color: #f8f8ff;
              color: #4b0082;
              font: bold 10pt &quot;Segoe UI&quot;;
              border: 1px solid #d8bfd8;
              border-radius: 8px;
              padding: 6px;
          }
         </string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
